### ISSUE TITLE
Request: Color Noise (Example attached)

### ISSUE THREAD

**Issue by DenisWASD** [ephtracy/ephtracy.github.io#88]
Hey!
It would be very cool, if you could select voxels and click button to randomize voxel colors to make a cool noise.
The color could be totally random or it could take the main Voxel color and then generate random brighter or darker colors 😄 

[Here's the example I found on Google:](https://thumbs.gfycat.com/GleamingKnobbyAmericanriverotter-poster.jpg)

**Comment by reneberwanger** [comment]
I do this with command rand, and with some shader plugins xs rand.

**Comment by DenisWASD** [comment]
Yes I know that this is possible, but I'd like to use a build-in one.

**Comment by jdiperla** [comment]
I support this request too.
